src/nats/nats.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { NatsStartupService } from './nats.startup';
+import { TriageModule } from '../triage/triage.module';
+import { LoggerService } from '@tazama-lf/frms-coe-lib';
+
+@Module({
+  imports: [TriageModule],
+  providers: [NatsStartupService, LoggerService],
+  exports: [NatsStartupService],
+})
```

Comment:
```suggestion
@Module({
  imports: [TriageModule, LoggerModule],
  providers: [NatsStartupService],
  exports: [NatsStartupService],
})
```
After the LoggerService changes the module files should look something like this.

Reply:
We have updated NatsModule to import LoggerModule instead of providing logggerservice directly. This ensures proper dependency injection and avoids multiple logger instances.